### DISCUSSION TITLE
Set the height for embedded integrations iframe

### DISCUSSION THREAD

**Discussion by JoeyJanson** [outline/outline#1524] [General]
I have never tried it before but just saw an embedded GitHub gist in our Outline. It was only 200px high and I couldn't read the code that well. I also couldn't find options to change the height. Is this possible? If not:
- Is it possible to add a dropdown in the editor after embedding something?
- Or a _**collapse**_ trigger right beneath the embed to show the full height embed?

**Comment by tommoor** [comment]
GitHub Gist's are unfortunately a weird beast, the way they do the embedding doesn't allow us to get the "native" height of the iframe so it's fixed to that 200px height as a workaround.

Given this restriction, do you think a button that for example doubled the height would be useful?

**Reply by JoeyJanson** [reply]
It would be better, yes. In this particular case it's a gist with 55 lines of code so personally I would prefer an expand option to set the height at 1200px.

**Reply by wforte88** [reply]
It would certainly be useful to be able to define the height of the embeds. I have several (Figma, Google Docs) that have information cut off due to the predefined height limit. If there were a way to customize the height of some of them, the experience of using the embeds would be much better.
